openmdao/docs/usr-guide/tutorials/beam-sizing.rst

```diff
@@ -0,0 +1,430 @@
+
+.. index:: MDAO tutorial problem
+
+======================
+Beam Sizing Problem
+======================
+
+Story Problem
+-----------
```

Comment:
here, and in just about every header you have, the underline is too short.  number of dashes must equal or exceed number of characters in the title above the dashes to correctly build.